docs/developer-guide/syntax-reference.rst

```diff
@@ -313,7 +313,7 @@ continuously write the result of the SELECT query into the stream and
 its corresponding topic.
 
 If the PARTITION BY clause is present, then the resulting stream will
-have the specified column as its key.
+have the specified column as its key. The specified column must be present in the query result.
```

Comment:
Would it better to explicitly reference to the `column_name` instead? like `have the column_name as its key` instead of `the specified column`? 

Btw, what is the query result? is it that the `column_name` must be in present or part of the `select_expr`?

Reply:
The `column_name` has to correspond to the name of one of the columns in the resulting stream. So yes, it has to be a part of the `select_expr`, but it has to be the actual name of the column.

Reply:
I agree with @spena's two points: (1) we should replace "specified column" with "\`\`column_name\`\`" and (2) we should mention that "must be present in the `select_expr`". Good catch clarifying the docs though!